src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_util.py

```diff
@@ -0,0 +1,190 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import register_cli_argument, cli_command
+from azure.cli.core.commands.parameters import ignore_type
+from azure.cli.core.commands.arm import cli_generic_update_command
+
+
+# MANAGEMENT CLIENT FACTORIES
+def get_monitor_management_client(_):
+    from azure.mgmt.monitor import MonitorManagementClient
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
+    return get_mgmt_service_client(MonitorManagementClient)
+
+
+def get_monitor_autoscale_settings_operation(kwargs):
+    return get_monitor_management_client(kwargs).autoscale_settings
+
+
+def get_monitor_service_diagnostic_settings_operation(kwargs):
+    return get_monitor_management_client(kwargs).service_diagnostic_settings
+
+
+def get_monitor_alert_rules_operation(kwargs):
+    return get_monitor_management_client(kwargs).alert_rules
+
+
+def get_monitor_alert_rule_incidents_operation(kwargs):
+    return get_monitor_management_client(kwargs).alert_rule_incidents
+
+
+def get_monitor_log_profiles_operation(kwargs):
+    return get_monitor_management_client(kwargs).log_profiles
+
+
+# DATA CLIENT FACTORIES
+def get_monitor_client(_):
+    from azure.monitor import MonitorClient
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
+    return get_mgmt_service_client(MonitorClient)
```

Comment:
strange...looks like i missed something here..Thanks for catching..Let me verify